tsconfig.json

```diff
@@ -11,7 +11,8 @@
     "typeRoots": [
       "./node_modules/@verdaccio/types/lib/verdaccio",
       "./node_modules/@types"
-    ]
+    ],
+    "experimentalDecorators": true
```

Comment:
I think, we should not use experimental stuff.

Reply:
I added this line to allow MSVC to use decorators (as https://ihatetomatoes.net/how-to-remove-experimentaldecorators-warning-in-vscode/ says).

Do not know why they call it "experimental" as decorators was added many years ago.

Looks like this this is not very experimental and we can use it.